src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesFolderProperty.java

```diff
@@ -260,6 +222,18 @@ public String getDisplayName() {
             return Messages.KubernetesFolderProperty_displayName();
         }
 
+        public static List<UsagePermission> getEffectivePermissions() {
```

Comment:
Shouldn't be static if you want to call it from jelly like this.

```suggestion
        @SuppressWarnings("unused") // Used by jelly
        @Restricted(DoNotUse.class) // Used by jelly
        public List<UsagePermission> getEffectivePermissions() {
```